src/Imdb/Calendar.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+#############################################################################
+# IMDBPHP                              (c) Giorgos Giagas & Itzchak Rehberg #
+# written by Giorgos Giagas                                                 #
+# extended & maintained by Itzchak Rehberg <izzysoft AT qumran DOT org>     #
+# http://www.izzysoft.de/                                                   #
+# ------------------------------------------------------------------------- #
+# IMDBPHP TOP CHARTS                                      (c) Ricardo Silva #
+# written by Ricardo Silva (banzap) <banzap@gmail.com>                      #
+# http://www.ricardosilva.pt.tl/                                            #
+# ------------------------------------------------------------------------- #
+# IMDBPHP UPCOMMING RELEASES (based on IMDBPHP TOP CHARTS (c)Ricardo Silva) #
+# written/modified/extended by Ed (github user: duck7000)                   #
+# ------------------------------------------------------------------------- #
+# This program is free software; you can redistribute and/or modify it      #
+# under the terms of the GNU General Public License (see doc/LICENSE)       #
+#############################################################################
+
+namespace Imdb;
+
+/**
+ * Obtaining information about upcomming movie releases as seen on imdB
+ * https://www.imdb.com/calendar
+ * @author Ricardo Silva (banzap) <banzap@gmail.com>
+ * @author Ed (github user: duck7000)
+ */
+class Calendar extends MdbBase
+{
+
+    protected $page = null;
+
+    /**
+     * Get upcomming movie releases as seen on imdB
+     * @return array (array[string date, array releases] of arrays[title,year,imdbid])
+     * @parameter $country, This defines wich country list is returned
+     * for example de, nl, us as they appear on https://www.imdb.com/calendar
+     */
+    public function upcommingReleases($country)
+    {
+        $page = $this->getXpathPage($country);
+        $calendar = array();
+        $dates = $page->query("//*[@id='main']/h4");
+        foreach ($dates as $key => $date) {
+            $key++;
+            $titlesRaw = $page->query("//*[@id='main']/ul[$key]//li");
+            $titles = array();
+            foreach ($titlesRaw as $value) {
+                $href = $value->getElementsByTagName('a')->item(0)->getAttribute('href');
+                preg_match('!.*?/title/tt(\d+)/.*!', $href, $imdbid);
+                $title = trim($value->getElementsByTagName('a')->item(0)->nodeValue);
+                preg_match('#\((.*?)\)#', $value->nodeValue, $year);
+                $titles[] = array(
+                    'title' => $title,
+                    'year' => $year[1],
+                    'imdbid' => $imdbid[1]
+                );
+            }
+            $calendar[] = array(
+                'date' => trim($date->nodeValue),
```

Comment:
Might be nice to parse this date to save the user the effort of figuring out and parsing this format.
Not sure though, the format doesn't seem to be locale specific, I get the same date strings whatever language i ask for .. which is good .. but if that changes it'll be hard to fix.
``` $d = DateTime::createFromFormat("d F Y", "06 April 2022")```

Reply:
This might be a good idea, unless we decide to leave out the dates?

Reply:
your suggestion does not work, there need to be a \ in front of DateTime
Otherwise DateTime is not found in the current namespace (i don't really no what namespaces are but found the solution online)

Reply:
I changed this locally as well, works great so i will change it in this PR

Reply:
This is added in PR